packages/ui/src/Modal.tsx

```diff
@@ -23,6 +23,7 @@ interface ModalProps {
   primaryButtonOnClick?: (value?: any) => void;
   // Renders a primary gray button to the left of the primary button in the footer, if no footer prop is provided.
```

Comment:
I think this comment needs to move down.

Reply:
Whoops, good catch! Fixed.